README.md

```diff
@@ -1,4 +1,58 @@
 # HopsonBot
-Discord bot for the Hopson Community Server, written using Discord.js
+HopsonBot is a Discord bot for the [Hopson Community Server](https://discordapp.com/invite/xvQaysR), written using Discord.js.
 
-> Put your config in the data folder.
```

Comment:
I don't think the bot is being used by any other server for now or in the foreseeable future.